test/browser_tests/spec_suites/shared/shared_office.js

```diff
@@ -0,0 +1,22 @@
+var Office = require( '../../page_objects/page_office.js' );
+
+describe( 'The offices template', function() {
+  it( 'should properly load in a browser', function() {
+    var page = new Office();
+    page.get( 'ProjectCatalyst' );
```

Comment:
I wonder if it's worth breaking up in the page objects like I did, but instead of repeating the ones that are shared across pages, there's a function that get's passed the parent element (like this.eventElem in my PR) and returns an object with the shared elems saved.